ocs_ci/ocs/warp.py

```diff
@@ -0,0 +1,186 @@
+import logging
+import os
+
+from ocs_ci.helpers import helpers
+from ocs_ci.utility import templating
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs import constants
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources import pod
+from tempfile import mkdtemp
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+
+log = logging.getLogger(__name__)
+
+
+class Warp(object):
+    """
+    Warp S3 benchmark
+
+    """
+
+    def __init__(self):
+        """
+        Initializer to create warp pod to running warp benchmark
+
+        """
+        self.pod_dic_path = constants.WARP_YAML
+        self.namespace = config.ENV_DATA["cluster_namespace"]
+        self.ocp_obj = OCP(namespace=self.namespace)
+        self.bucket_name = None
+        self.warp_cr = templating.load_yaml(constants.WARP_OBJ_YAML)
+        self.host = self.warp_cr["host"]
+        self.duration = self.warp_cr["duration"]
+        self.concurrent = self.warp_cr["concurrent"]
+        self.objects = self.warp_cr["objects"]
+        self.obj_size = self.warp_cr["obj.size"]
+        self.get_distrib = self.warp_cr["get-distrib"]
+        self.put_distrib = self.warp_cr["put-distrib"]
+        self.delete_distrib = self.warp_cr["delete-distrib"]
+        self.stat_distrib = self.warp_cr["stat-distrib"]
+        self.warp_bin_dir = self.warp_cr["warp_bin_dir"]
+        self.output_file = "output.csv"
+        self.warp_dir = mkdtemp(prefix="warp-")
+
+    def create_resource_warp(self):
+        """
+        Create resource for Warp S3 benchmark:
```

Comment:
it would be a good idea to add a link to the warp benchmark page here. 
`https://github.com/minio/warp`
Also, a high-level explanation of what running workload means.

Reply:
Fixed.  Added more description in Warp class.